README.md

```diff
@@ -247,6 +245,10 @@ The structure should look like this:
 ```
 
 
+When the `devMode` is enabled. You will see information boxes at the address the server is running on.
+There is described which component you have to implement and which information you will be getting.
```

Comment:
"There it is described" sounds better than "There is described" in this context.